adopters.md

```diff
@@ -0,0 +1,107 @@
+Who is using Flatcar?
+====================
+
+Sharing experiences and learning from other users is essential. While the [Flatcar Slack community](https://kubernetes.slack.com/archives/C03GQ8B5XNJ) allows
+users to get in touch and also [the Matrix channel](https://app.element.io/#/room/#flatcar:matrix.org) The following is a directory of adopters to help identify users of individual features. The users themselves directly maintain the list.
+
+
+Adding yourself as a user
+-------------------------
+
+If you are using Flatcar, please consider adding yourself as a user with a quick description of your use case by opening a pull request to this file and adding a section describing your usage of Flatcar. If you are open to others contacting you about your use of Flatcar on Slack or Matrix, add your Slack/Matrix nickname as well.
+    
+    * N: Name of user (company or individual) 
+      D: Description 
+      L: Link with further information (optional) 
+      C: Contacts available for questions (optional)
+
+Example entry:
+
+    * N: Flatcar Example User Inc.
+      D: Flatcar Example User Inc. is using Flatcar 
+      L: https://www.flatcar.org/ 
+      C: @slacknick and/or @Matrixnick 
+    
+
+Requirements to be listed
+-------------------------
+
+ * You must represent the user listed. Do not add entries on behalf of
+   other users.
+* Commercial or production use is not required a user can be an end user, cloud service provider or consultant as long as it is permanent deployment and not a trail deployment. A well-done home lab setup can be equally
+   interesting as a large-scale commercial deployment.
+
+
+Users 
+-----
+ 
+    * N: Adobe
+    
+    * N: AloPeyk
+      D: We are an on-demand delivery business that every day serve millions of requests on a consistent bare metal infrastructure have this concern to choose a reliable and atomic operations system but none of the common OS couldn't satisfy us for such heavy workloads. Since we migrated our production Kubernetes cluster nothing can break this consistent and solid cluster which is powered by amazing Flatcoar OS.
+    
+    * N: Cloud house
+      D: Flatcar OS has been useful for us for our on-premise solution to our customers 
+      
+    * N: Equinix Metal
+
+      
+    * N: Finleap Connect
+      D: Finleap Connect - At finleap connect we serve over a million financial transactions per day. As a regulated company using a pure cloud-native stack based on Kubernetes, using Flatcar as our foundational building block for reliable, secure and immutable nodes across the public clouds and on bare-metal deployments was a day one decision we never regretted. Today Flatcar serves all of our 12 production clusters with over 300 nodes on public clouds in 3 countries and on our bare-metal private-cloud setup.
+    
+    * N: Host with quantum 
+      C: https://twitter.com/klimpong
```

Comment:
Can I PR some changes to this? 😆

Reply:
Yes please do!!

Reply:
@till please feel free to [suggest](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) right here in the PR if that works for you!